sopel/modules/isup.py

```diff
@@ -5,12 +5,13 @@
 
 from sopel import web
 from sopel.module import commands
+from requests.exceptions import SSLError
 
-
```

Comment:
Keep this blank line or flake8 will complain. (Soon, the test suite will enforce this for you; see #1278.)